### PR TITLE
Adding new polkadot parachains

### PR DESCRIPTION
**Pull request by stepanLav**
## Efinity
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-15 at 11 08 59

Screenshot 2022-03-15 at 11 09 27

</details>

## Centrifuge
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-15 at 11 11 02

Screenshot 2022-03-15 at 11 11 31

</details>

### This network is blocking connection to wss://rpc-01.hydradx.io by Origin header which must equal https://polkadot.js
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-15 at 11 12 16

Screenshot 2022-03-15 at 11 12 28

Screenshot 2022-03-15 at 11 13 00

</details>

## Interlay
<details>
  <summary>Spoiler!</summary>
Screenshot 2022-03-15 at 11 14 11

Screenshot 2022-03-15 at 11 14 28

Screenshot 2022-03-15 at 11 14 53

</details>

Interlay tokens: 
https://github.com/interlay/interbtc/blob/master/primitives/src/lib.rs#L473

Interlay existential deposit:
https://github.com/interlay/interbtc/blob/34c24961384208d8a22cef3446fea22d585c3ef7/parachain/runtime/interlay/src/lib.rs#L942

### CODE REVIEW

**Review by valentunn — CHANGES_REQUESTED** [review]
Interlay uses orml

**Comment by stepanLav** [comment]
> Interlay uses orml

fixed!

File: chains/v2/chains_dev.json [hunk 1]
```diff
@@ -2134,5 +2134,189 @@
...
+                "typeExtras": {
+                    "currencyIdScale": "0x0200",
+                    "currencyIdType": "interbtc_primitives.CurrencyId",
+                    "existentialDeposit": "0",
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 1,
+                "symbol": "INTERBTC",
```
[the remark this answers is not in the document]
Comment (antonkhvorov) [reply chains/v2/chains_dev.json:2244]: yes, I agree — interBTC is the right name based on Interlay website

File: chains/v2/chains_dev.json [hunk 2]
```diff
@@ -2134,5 +2134,191 @@
...
+                "typeExtras": {
+                    "currencyIdScale": "0x0200",
+                    "currencyIdType": "interbtc_primitives.CurrencyId",
+                    "existentialDeposit": "0",
+                    "transfersEnabled": true
+                }
+            },
+            {
+                "assetId": 1,
+                "symbol": "interBTC",
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:2244]: "priceId": "bitcoin"

Comment (stepanLav) [reply chains/v2/chains_dev.json:2244]: fixed

**Review by antonkhvorov — APPROVED** [review]

**Comment by stepanLav** [comment]
> Interlay uses orml

Could you resolve your change request?

**Review by valentunn — APPROVED** [review]

**Pull request merged**